### DISCUSSION TITLE
same visitor id on two different device in iphone 13

### DISCUSSION THREAD

**Discussion by chandreshGSCT** [fingerprintjs/fingerprintjs#971] [General]
i have get visitor id but issue was in my iphone and my friend iphone are getting same visitor id

**Comment by JuroUhlar** [comment]
Hi @chandreshGSCT, thanks for the report, could you provide us with the visitor ID? We will investigate what happened, thank you!

**Comment by chandreshGSCT** [comment]
here is my fingerprint "81ff4856e4822e81c8e3ff43c3869052"

i have used in chrome and my mobile model iPhone 13, 128GB updated OS and Browser Version (Both device is same)

**Reply by JuroUhlar** [reply]
Oh, I see, sorry about the confusion, you are talking about FingerprintJS, we cannot investigate much there. FingerprintJS uses a simple identification mechanism that collects browser attributes and hashes them into the identifier you see. You can examine all the signals used here: https://fingerprintjs.github.io/fingerprintjs/

For highly uniform devices like iPhones running the same browser version, with the same language settings, it is expected to give the same browser identifier as all the browser attributes are the same. This is in contrast to Fingerprint Pro which collects more attributes, including server-side information, and processes them using a variety of advanced matching techniques to give unique visitor IDs even to highly uniform browser configurations. The accuracy of FingerprintJS is around 60% compared to Fingerprint Pro's 99.5%. You can see the full comparison here: https://fingerprint.com/github/
